server/odc-service/src/main/java/com/oceanbase/odc/service/connection/ConnectionHelper.java

```diff
@@ -38,12 +35,9 @@
 @Validated
 public class ConnectionHelper {
 
-    @Autowired
-    private SensitivePropertyHandler sensitiveHandler;
-
     public ConnectionStringParseResult parseConnectionStr(@NotNull @Valid ParseConnectionStringReq req) {
         ConnectionStringParseResult result = MySQLClientArgsParser.parse2(req.getConnStr());
-        result.setPassword(sensitiveHandler.encrypt(result.getPassword()));
+        result.setPassword(null);
```

Comment:
can we just remove password field , or mark the password filed as WRITEONLY ?

Reply:
Yes we can. btw, this api is deprecated and will be deleted after this merge.